### ISSUE TITLE
Mimicry, lateral shift and befuddle interaction

### ISSUE THREAD

**Issue by Rikuns** [keyteki/keyteki#3962]
Opponent played Befuddle chosing Untamed.
I then called Untamed and played Mimicry, chosing Lateral Shift.
At this point I understand since opponent deck doesnt have Untamed, i wont be able to play anything.
But TCO just skip the whole effect and not even let me see opponent hand. I believe I should still look at opponent hand.

**Comment by dmikalova** [comment]
I don't believe this is true - looking at the Mimicry FAQ you cannot play alpha cards after your first card bc Mimicry is a copy of the card - and I would extend that to Befuddle, where bc Mimicry is a copy of a non-untamed lateral shift you cannot play the lateral shift, or anything else.

It's not clear that Mimicry with cannot play would behave the same as alpha - ie alpha would put mimicry back to hand, but maybe cannot fizzles mimicry and it goes to discard. In this situation I think the resulting game state is effectively the same.
